### PR TITLE
README: Add information about xclip

### PR DESCRIPTION
**Pull request by htfy96**

### CODE REVIEW

**Comment by htfy96** [comment]
I signed it.

**Comment by chiphogg** [comment]
Sorry I'm just getting to look at this now.  I've been super busy with my new job.

Speaking of which, it means I no longer have access to merge pull requests here!  Somebody else who still works at Google will have to do that.  :)

**Comment by dbarnett** [comment]
Hmm, forgot about that. I can merge it if it looks good to you.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,14 @@ Syncopate is expected to work on any platform vim supports, but the default
 configuration may not work on your system and direct-to-clipboard support hasn't
 been implemented for some platforms. Contributions welcome!
 
+### xclip
+
+Syncopate requires `xclip` to manipulate clipboard. In most cases installing it from your package manager should just work, but for Arch users the `xclip` in official repository(0.12.4) is too old and doesn't support `-target`. They should install `xclip-svn` from AUR:
```
Comment (chiphogg) [inline README.md]: Replace this line with the following:

```
Syncopate requires `xclip` to manipulate the clipboard.  In most cases,
installing it from your package manager should just work.

Arch Linux's official repository has an `xclip` (0.12.4) which is too old: it
doesn't support `--target`. Arch users should install `xclip-svn` from AUR:
```

I tweaked the grammar a bit, and wrapped the lines.

**Comment by htfy96** [comment]
Done.

**Comment by chiphogg** [comment]
LGTM.

Thanks for pointing this out and helping us fix it!

**Pull request merged**